src/Workspaces/Core/Portable/Rename/ConflictEngine/ConflictResolver.Session.cs

```diff
@@ -30,7 +30,7 @@ internal static partial class ConflictResolver
         private class Session
         {
             // Set of All Locations that will be renamed (does not include non-reference locations that need to be checked for conflicts)
-            private readonly RenameLocations _renameLocationSet;
+            private readonly HeavyweightRenameLocations _renameLocationSet;
```

Comment:
this is the component that actually does conflit resolution.  It lives in OOP and actually wants to work with and process symbols/compilations/etc.  as such, it holds onto a HeavyweightRenameLocations instance.